openedx/features/offline_mode/urls.py

```diff
@@ -0,0 +1,10 @@
+"""
+URLs for the offline_mode feature.
+"""
+from django.urls import path
+
+from .views import SudioCoursePublishedEventHandler
+
+urlpatterns = [
+    path('handle_course_published', SudioCoursePublishedEventHandler.as_view(), name='handle_course_published'),
```

Comment:
It's definitely required to write up rationale for a separate Open edX application for offline support, as I believe we would have hard time merging it without ADR unfortunately.